modules/certificate/certificate.resolver.js

```diff
@@ -0,0 +1,31 @@
+const certificatesService = require('./certificate.service');
+const RuleError = require('../../errors/rule.error');
+
+const tryCatchWrapper = async func => {
```

Comment:
I guess its not pretty good naming. Mb better `withErrorHandling` ? and call param `handler`